src/sidebar/index.js

```diff
@@ -10,20 +10,15 @@ require('../shared/polyfills');
 var raven;
 
 // Read settings rendered into sidebar app HTML by service/extension.
-var settings = require('../shared/settings').jsonConfigsFrom(document);
+var appConfig = require('../shared/settings').jsonConfigsFrom(document);
```

Comment:
I renamed this to clarify that it refers to the config rendered into the sidebar app's HTML page.